functable.c

```diff
@@ -608,6 +600,8 @@ Z_INTERNAL uint32_t longest_match_slow_stub(deflate_state *const s, Pos cur_matc
     if (x86_cpu_has_avx2)
         functable.longest_match_slow = &longest_match_slow_unaligned_avx2;
 #  endif
+#else
```

Comment:
This, and the above functions too, do not have the _c function as a fallback in the UNALIGNED_OK path.
As far as I can see, this would remain unset if compiled without builtin_ctz and run on an 486 cpu for example.

Reply:
It would use the `longest_match_slow_unaligned_16` function it doesn't need to use `longest_match_slow_c`.

Reply:
Of course, no idea how I missed that. 🤦‍♂️